README.md

```diff
@@ -217,6 +217,9 @@ Para mais informações acesse de como criar o Access Token e configurar veja os
 - [How to Create a Personal Access Token](https://www.digitalocean.com/docs/apis-clis/api/create-personal-access-token)
 - [Terraform - Digital Ocean Provider](https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs)
 
+E para mais informações a respeito do SSH Fingerprint:
+- [How-to Add SSH Keys to New or Existing Droplets](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/)
```

Comment:
Added this link so people can find more information about the SSH Fingerprint used along the `README.md`.